README.md

```diff
@@ -25,28 +28,38 @@ zplug "greymd/ttcopy"
 
 ## zsh, bash
 
+#### 1. Clone Repository
+
+```sh
+$ git clone https://github.com/greymd/ttcopy.git ~/ttcopy
+```
+
+#### 2. Edit `.bashrc` or `.zshrc`
+
+And add following lines.
+
 ```sh
 # Set ID and PASSWORD **AS YOU LIKE**.
```

Comment:
Well, though not scope in this commit, perhaps user will get confused to set identity. We should note

 1. User do not require to register. (the reason of why they set choose ID/Password "as they like")
 2. So the user should try to keep the id unique.

Can you add some lines?